src/components/config/SecureTextArea.tsx

```diff
@@ -32,7 +32,7 @@ export const SecureTextArea = ({
         </div>
       ) : (
         <div className="gf-form gf-form--grow">
```

Comment:
you mentioned removing the legacy forms, should this also be removed?
```suggestion
        <div>
```